scripts/run_lint_python.sh

```diff
@@ -34,5 +34,5 @@ display_result $? 1 "isort linting check"
 ./scripts/run_mypy.sh
 display_result $? 1 "Mypy type check"
 
-black --check . --exclude node_modules
+black --check . --exclude node_modules --exclude venv
```

Comment:
I think this change should be reverted, sorry If I caused confusion, I'd meant that if you use pipenv to make the virtualenv outside of the main directory then this wouldn't be an issue anymore

Reply:
No worries, I can remove those